docs/testing/playwright.mdx

```diff
@@ -273,4 +273,101 @@ test('ensure that clerk has loaded', async ({ page }) => {
 })
 ```
 
+## Testing authenticated flows
+
+Playwright executes tests in isolated environments called browser contexts. So, each test case runs in a new browser context, which means that the user session is not shared between test cases. However, tests can load existing authenticated state in Playwright.
+
+Following the [Playwright docs on Authentication](https://playwright.dev/docs/auth), you can test authenticated flows without having to sign in for each test case.
+
+The steps below showcase an example of how to save the session storage in a file and load it in your test cases. Of course, you can customize this to fit your needs. You can also have a look at our example repository [here](https://github.com/clerk/clerk-playwright-nextjs).
+
+<Steps>
+  ### Add the storage file in your `.gitingore`
+
+  Create a `playwright/.clerk` directory and add it to your `.gitignore`.
```

Comment:
It looks like this section, https://playwright.dev/docs/auth#core-concepts, explains: "We recommend to create playwright/.auth directory and add it to your .gitignore. Your authentication routine will produce authenticated browser state and save it to a file in this playwright/.auth directory. Later on, tests will reuse this state and start already authenticated."

I'm assuming that is what the `playwright/.clerk` file is going to do, but I just want to be sure.

Reply:
Yes, that's right! I just renamed the directory to `.clerk` because it includes Clerk-related stuff. The name of the directory does not really matter.